src/components/Initiatives/InitiativesPage/InitiativesPage.tsx

```diff
@@ -79,7 +80,7 @@ export const InitiativesPage = () => {
   const cortexApi = useApi(cortexApiRef);
 
   const {
-    value: initiatives,
+    value: scorecards,
```

Comment:
Is this right? Isn't this the initiatives page?

Reply:
😆 I went through the same denial when debugging the test. This matches this fetch:
https://github.com/cortexapps/backstage-plugin/blob/fabee9bda4fc5e6dd4a3569d4dddf0dd6f2ee47b/src/components/Initiatives/InitiativesPage/InitiativesPage.tsx#L86

Confusingly, there's a nested component in the same file above that is actually fetching initiatives...
https://github.com/cortexapps/backstage-plugin/blob/fabee9bda4fc5e6dd4a3569d4dddf0dd6f2ee47b/src/components/Initiatives/InitiativesPage/InitiativesPage.tsx#L36-L42

Reply:
😆